### PR TITLE
Added more model for training the dataset

### LINKED ISSUE #41: Adding more model in model training  for better prediction
We can more model like xgboost , cat boost and lightgbm model for better accuracy in .ipynb file.
The code can be added here in model training:
image

### PR DESCRIPTION
**Pull request by Prasadayus**
Fixes #41 . Added the xgboost , lightgbm and xgboost model for training the dataset

Discod id: PAyush#3770

### CODE REVIEW

**Comment by BamaCharanChhandogi** [comment]
Thank you for submitting your pull request! 🙌 We'll review it as soon as possible. In the meantime, please ensure that your changes align with our [CONTRIBUTING.md](https://github.com/BamaCharanChhandogi/Diabetes-Prediction/blob/main/CONTRIBUTING.md). If there are any specific instructions or feedback regarding your PR, we'll provide them here. Thanks again for your contribution! 😊

**Comment by BamaCharanChhandogi** [comment]
I will see it later.
